### ISSUE TITLE
Few design questions while I implement the LESS version...

### ISSUE THREAD

**Issue by dharapvj** [jquery-ui-bootstrap/jquery-ui-bootstrap#205]
### Main Design Questions
1. **Non-variable colors** - @gtraxx - I am looking at the `jquery-ui-1.10.3.custom.css` in the masterbs3 branch and it contains all sorts of color references which are not present in bootstrap3 default theme. e.g. `.ui-widget-header` definition on [L262](https://github.com/addyosmani/jquery-ui-bootstrap/blob/masterbs3/css/custom-theme/jquery-ui-1.10.3.custom.css#L262). Now .. should I consider these are work-in-progress OR should I plan on adding these are new color definitions in my LESS files?
2. **Browser Support**
   _Background_: If we look at [bootstrap3 browser support](http://getbootstrap.com/getting-started/#browsers), they have dropped older browser support for pretty much every browser. Gone with that are some of the LESS `mixins` which they used to use. e.g. [`.border-radius`](https://github.com/twbs/bootstrap/blob/v2.3.2/less/mixins.less#L205) to provide cross browser border radius. Now if I check [border radius support](http://caniuse.com/#search=border-radius) by different browsers, it is supported by all latest browsers and hence BS3 guys are consistent in their stance of dropping the said mixin from their LESS files. 
   _Question_: Now my questions is - what is our stance for older browser support? Are we going to go with BS3 stance or we replicate whatever that is dropped from latest bootstrap and continue to provide support for older browsers?
### META Question

For questions like these - github does not provide any discussion area. So even when this was not even a issue, I had to open a issue to you guys. Is there anything else that can be used for such meta design discussions? May be [trello](https://trello.com/)?

I have other points to discuss too.. e.g. 
- using a simple express based node server to allow people to host this content
- moving to a Grunt based distribution creation, 
- seperating the gh-pages from core css so that those who want css, can simply get the css download only via "master" branch - this will need grunt and travis build.
- providing the LESS designer -this will need a build too, etc

But these are all "design" type questions.. which needs discussion among core implementers. They are not really issues since we may never get energy to implement those.. So I find it hard to call them out as issues.

What are your thought on place for such discussions?

**Comment by gtraxx** [comment]
#### Non-variable colors

Some colors are not present in bootstrap 3 but I used the color panel-heading for ui-widget-header

#### Browser Support

I continue support for old browsers but with a limit and I used a "jquery-ui-1.10.3.theme.css" style sheet for the gradient as in Bootstrap 3.

**Comment by dharapvj** [comment]
Hi @gtraxx 

Thanks for the clarifications.

Could you point me to line number of this color panel heading that you mentioned above?

Regarding the older browsers - I think I am following the content of "jquery-ui-1.10.3.custom.css" since that was the only one added in the index.html.

So, may be there will be two versions of some LESS stylessheet files too.

**Comment by gtraxx** [comment]
Hi, One should compare the CSS version with the Less version and check the files (see the differences)
Thanks :)

**Comment by dharapvj** [comment]
Let me do that and report back to you.. 

That's what you meant right?

**Comment by gtraxx** [comment]
yes, I could edit the CSS file and use Less to create the file. 
I'll edit the documentation with new features :)
Thanks
